publisher/publisher.go

```diff
@@ -242,7 +242,7 @@ func (pub *Impl) SubmitToSingleCTWithResult(ctx context.Context, req *pubpb.Requ
 			return nil, err
 		}
 		var body string
-		if respErr, ok := err.(ctClient.RspError); ok && respErr.StatusCode < 500 {
```

Comment:
This is required because the upstream dep moved the `RspError` into the `jsonclient` package.